### PR TITLE
Markdown catch if not real ID

### PR DESCRIPTION
**Pull request by auravoid**

### CODE REVIEW

File: Commands/Markdown.cs [hunk 1]
```diff
@@ -24,7 +24,19 @@ public class Markdown : ApplicationCommandModule
                 return;
             }
 
-            message = await ctx.Channel.GetMessageAsync(messageId);
+            DiscordChannel channel;
+            channel = await ctx.Client.GetChannelAsync(ctx.Channel.Id);
```
Comment (FloatingMilkshake) [inline Commands/Markdown.cs:28]: This is redundant and just makes the bot send a request to fetch a channel it already has in memory (`ctx.Channel`)

File: Commands/Markdown.cs [hunk 2]
```diff
@@ -24,7 +24,19 @@ public class Markdown : ApplicationCommandModule
...
+            
+            try
+
```
Comment (FloatingMilkshake) [inline Commands/Markdown.cs:31]: pls remove this newline it bothers me

**Review by FloatingMilkshake — CHANGES_REQUESTED** [review]

File: Commands/Markdown.cs (hunk 2 above)
Comment (auravoid) [reply Commands/Markdown.cs:31]: no

File: Commands/Markdown.cs [hunk 3]
```diff
@@ -23,8 +23,17 @@ public class Markdown : ApplicationCommandModule
                     "Hmm, that doesn't look like a valid message ID or link. I wasn't able to get the Markdown data from it."));
                 return;
             }
-
-            message = await ctx.Channel.GetMessageAsync(messageId);
+            
+            try
+            // Do not delete
```
Comment (FloatingMilkshake) [inline Commands/Markdown.cs:28]: smh

Comment (auravoid) [reply Commands/Markdown.cs:28]: no

**Review by FloatingMilkshake — COMMENTED** [review]
you can thank GitHub's web editor for this lol

**Comment by auravoid** [comment]
> you can thank GitHub's web editor for this lol

no

**Review by FloatingMilkshake — APPROVED** [review]
looks good to me, thanks for the fix

**Comment by auravoid** [comment]
> looks good to me, thanks for the fix

no

**Pull request merged**